experiments/react-native-uifabric/metro.config.js

```diff
@@ -0,0 +1,54 @@
+// @ts-check
+
+const fs = require('fs');
+const path = require('path');
+const blacklist = require('metro-config/src/defaults/blacklist');
+const getAllPackageInfo = require('../../scripts/monorepo/getAllPackageInfo');
+
+// Helper function to get the real path to a symlinked package by searching for its package.json
+const pathToSymLinkedPackage = package => fs.realpathSync(path.resolve(require.resolve(package + '/package.json'), '..'));
+
+// Get an array of all packages under repo-root/packages
+const allPackages = getAllPackageInfo();
+const uifabricsharedPackages = Object.keys(allPackages).reduce(
+  (prev, key) => (allPackages[key].packagePath.startsWith('packages') ? [...prev, key] : prev),
+  []
+);
+
+const additionalWatchFolders = uifabricsharedPackages.map(package => pathToSymLinkedPackage(package));
+
+const extraNodeModules = {
+  'react-native': path.resolve(__dirname, '../../node_modules/@office-iss/react-native-win32')
+};
```

Comment:
Can we just inline this?  seems odd to have it separate.

Reply:
ok.